### PR TITLE
MSlice Fixes for 3.13

### PR DESCRIPTION
**Pull request by martyngigg**
**Description of work.**

Bring in minor fixes for mslice.

**To test:**

* Start mslice from the `Interfaces/Direct` menu
* Play with slices/cuts

*No associated issue*

Does this update require release notes?
- [ ] Yes
- [X] No

---

#### Reviewer ####

Please comment on the following ([full description](http://www.mantidproject.org/Individual_Ticket_Testing)):

##### Code Review #####

- [ ] Is the code of an acceptable quality?
- [ ] Does the code conform to the [coding standards](http://www.mantidproject.org/Coding_Standards)?
- [ ] Are the unit tests small and test the class in isolation?
- [ ] If there are changes in the release notes then do they describe the changes appropriately?

##### Functional Tests #####

- [ ] Do changes function as described? Add comments below that describe the tests performed?
- [ ] Do the changes handle unexpected situations, e.g. bad input?
- [ ] Has the relevant (user and developer) documentation been added/updated?

Does everything look good? Mark the review as **Approve**. A member of `@mantidproject/gatekeepers` will take care of it.

### CODE REVIEW

**Comment by NickDraper** [comment]
@MatthewBowles You are the best person to test this, do you have some time today?

**Comment by samueljackson92** [comment]
Another issue found from unscripted testing. https://github.com/mantidproject/mslice/issues/342

**Comment by mducle** [comment]
The release notes should also say that now MSlice workspaces are not stored in the `ADS` so they are invisible from MantidPlot. As some people may want to fit 1D cuts produce by MSlice using the MantidPlot algorithm, there should be a way to export an MSlice workspace to the `ADS` - which can be addressed as part of https://github.com/mantidproject/mslice/issues/341

File: scripts/ExternalInterfaces/CMakeLists.txt [hunk 1]
```diff
@@ -6,7 +6,7 @@ set ( _mslice_external_root ${CMAKE_CURRENT_BINARY_DIR}/mslice )
 ExternalProject_Add ( mslice
   PREFIX ${_mslice_external_root}
   GIT_REPOSITORY "https://github.com/mantidproject/mslice.git"
-  GIT_TAG 01292c88ccc508acb16735aa5f96450e076161bb
+  GIT_TAG a1c96d5d325b30e2537c5784a133c5284a936468
```
Comment (AndreiSavici) [inline scripts/ExternalInterfaces/CMakeLists.txt]: Should be ce0a3647d111bc31fafd00addb64913760828352

**Review by AndreiSavici — CHANGES_REQUESTED** [review]

File: scripts/ExternalInterfaces/CMakeLists.txt (hunk 1 above)
Comment (AndreiSavici) [reply scripts/ExternalInterfaces/CMakeLists.txt]: Or later, if more PRs are merged in

**Review by AndreiSavici — APPROVED** [review]

**Comment by AndreiSavici** [comment]
I've approved the changes. It might need to be modified again, if https://github.com/mantidproject/mslice/issues/347 is addressed.

**Comment by martyngigg** [comment]
@MatthewBowles Would you be able to add some release notes to this branch?

**Pull request merged**